danielforkner.Calculator/CalculatorLibrary/CalculatorLibrary.cs

```diff
@@ -0,0 +1,95 @@
+﻿using Newtonsoft.Json;
+using System.Linq;
+
+namespace CalculatorLibrary
+{
+    public class Calculation
+    {
+        public int Id { get; set; }
+        public double Operand1 { get; set; }
+        public double Operand2 { get; set; }
+        public string Operation { get; set; }
+        public double Result { get; set; }
+        public bool Success { get; set; }
+    }
```

Comment:
Add an empty line between methods and between classes

Reply:
Does this apply to fields and properties as well?

Reply:
@danielforkner no, just methods and classes. It's also good to have some separation between related blocks of code within a method so it doesn't become spaghetti 😁